examples/tests/sdk_tests/bucket/list.w

```diff
@@ -0,0 +1,21 @@
+bring cloud;
+
+let b = new cloud.Bucket();
+
+test "list" {
+  let var testArray = MutArray<str>[];
+
+  assert(b.list().length == 0);
+
+  b.put("file1.txt", "Foo");
+  b.put("file2.txt", "Bar");
+
+  testArray = b.list().copyMut();
+
+  assert(b.get(testArray.at(0)) == "Foo");
+  assert(b.get(testArray.at(1)) == "Bar");
```

Comment:
I think testing the keys is enough (otherwise we'll actually test `b.get`) :) 

you can take a look here:
examples/tests/valid/bucket_keys.w
https://github.com/winglang/wing/blob/0ff799d689555f697c0ffcacf0aae48c2f53bf5e/examples/tests/valid/bucket_keys.w#L1-L17